watchtower/wtdb/client_db_test.go

```diff
@@ -809,18 +816,30 @@ func TestClientDB(t *testing.T) {
 						err)
 				}
 
-				db, err := wtdb.OpenClientDB(
-					path, kvdb.DefaultDBTimeout,
-				)
+				bdb, err := wtdb.NewBoltBackendCreator(
+					true, path, wtdb.ClientDBName,
+				)(dbCfg)
+				if err != nil {
+					os.RemoveAll(path)
+					t.Fatalf("unable to open db: %v", err)
+				}
+
+				db, err := wtdb.OpenClientDB(bdb)
 				if err != nil {
 					os.RemoveAll(path)
 					t.Fatalf("unable to open db: %v", err)
 				}
 				db.Close()
 
-				db, err = wtdb.OpenClientDB(
-					path, kvdb.DefaultDBTimeout,
-				)
+				bdb, err = wtdb.NewBoltBackendCreator(
```

Comment:
Scope creep and could be done in a separate PR but we could also test these with etcd.

Reply:
I'll keep that in mind for a follow-up PR :+1: